### DISCUSSION TITLE
How to make custom URL schemes work with VSCode?

### DISCUSSION THREAD

**Discussion by supersetapp** [parcel-bundler/parcel#7084] [Q&A] [answered]
Using Parcel2.0.0, if I do a URL scheme like in [this example](https://parceljs.org/features/bundle-inlining/#inlining-a-bundle-as-text):

```typescript
import cssText from 'bundle-text:./test.css';
```

then Visual studio code gives me the error:

> Cannot find module 'bundle-text:./test.css' or its corresponding type declarations.ts(2307)

Is there any way to make this pattern play well with VSCode or other IDEs? It seems like I should instead use an actual path and then a transformer plugin if I want to do something different with the import?

**Comment by devongovett** [comment, marked as answer]
https://parceljs.org/features/dependency-resolution/#configuring-other-tools

**Reply by supersetapp** [reply]
Cool, thx! I had overlooked using a declarations file to declare a module for a scheme, that’s really helpful:

```typescript
declare module 'bundle-text:*' {
  const value: string;
  export default value;
}
```
